examples/jsm/shaders/MMDToonShader.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * MMD Toon Shader
+ *
+ * This shader is extended from MeshPhongMaterial, and merged algorithms with
+ * MeshToonMaterial and MeshMetcapMaterial.
+ * Ideas came from https://github.com/mrdoob/three.js/issues/19609
+ *
+ * Combining steps:
+ *  * Declare matcap uniform.
+ *  * Add gradientmap_pars_fragment.
+ *  * Combine dotNL and gradient irradiances as total irradiance.
```

Comment:
I guess this comment `Combine dotNL and gradient irradiances as total irradiance` is outdated?
Because irradiances are calculated only from gradient now.

Reply:
yes